libs/affinity/src/parse_affinity_options.cpp

```diff
@@ -112,56 +105,42 @@ namespace hpx { namespace threads { namespace detail
-                |   partlit("numa-balanced") >> qi::attr(numa_balanced)
-                ;
+            distribution = partlit("compact") >> qi::attr(compact) |
+                partlit("scatter") >> qi::attr(scatter) |
+                partlit("balanced") >> qi::attr(balanced) |
+                partlit("numa-balanced") >> qi::attr(numa_balanced);
 
-            thread_spec =
-                    partlit("thread") >> ':'
-                    >>  specs >> qi::attr(spec_type::thread)
-                ;
+            thread_spec = partlit("thread") >> ':' >> specs >>
+                qi::attr(spec_type::thread);
 
-            pu_spec =
-                    socket_spec >> core_spec >> processing_unit_spec
-//                 |   '~' >> pu_spec
+            pu_spec = socket_spec >> core_spec >> processing_unit_spec
+                //                 |   '~' >> pu_spec
                 ;
 
-            socket_spec =
-                    partlit("socket") >> ':'
-                    >> specs >> qi::attr(spec_type::socket)
-                |   partlit("numanode") >> ':'
-                    >> specs >> qi::attr(spec_type::numanode)
-                |   qi::attr(spec_type::unknown)
-                ;
+            socket_spec = partlit("socket") >> ':' >> specs >>
+                    qi::attr(spec_type::socket) |
+                partlit("numanode") >> ':' >> specs >>
+                    qi::attr(spec_type::numanode) |
+                qi::attr(spec_type::unknown);
 
-            core_spec =
-                   -qi::lit('.') >> partlit("core") >> ':'
-                    >> specs >> qi::attr(spec_type::core)
-                |   qi::attr(spec_type::unknown)
-                ;
+            core_spec = -qi::lit('.') >> partlit("core") >> ':' >> specs >>
+                    qi::attr(spec_type::core) |
+                qi::attr(spec_type::unknown);
 
-            processing_unit_spec =
-                   -qi::lit('.') >> partlit("pu") >> ':'
-                    >> specs >> qi::attr(spec_type::pu)
-                |   qi::attr(spec_type::unknown)
-                ;
+            processing_unit_spec = -qi::lit('.') >> partlit("pu") >> ':' >>
+                    specs >> qi::attr(spec_type::pu) |
+                qi::attr(spec_type::unknown);
 
             specs = spec % ',';
 
-            spec =
-                    qi::uint_ >> -(qi::int_)
-                |   partlit("all") >> qi::attr(spec_type::all_entities())
-                ;
+            spec = qi::uint_ >> -(qi::int_) |
+                partlit("all") >> qi::attr(spec_type::all_entities());
 
             BOOST_SPIRIT_DEBUG_NODES(
-                (start)(mapping)(distribution)(thread_spec)(pu_spec)(specs)
-                (spec)(socket_spec)(core_spec)(processing_unit_spec)
-            );
+                (start)(mapping)(distribution)(thread_spec)(pu_spec)(specs)(
+                    spec)(socket_spec)(core_spec)(processing_unit_spec));
         }
```

Comment:
Could we please not reformat the spirit parsers?

Reply:
My bad. Yes, definitely don't want to reformat them.